server/controllers/ItemMockCtrl.js

```diff
@@ -0,0 +1,91 @@
+const R = require('ramda');
+
+const ItemMockCtrl = (() => {
+  const categories = ['A', 'B', 'C', 'D'];
+  const validateCategoryOrNil = (value) => {
+    return R.or(R.isNil(value), R.includes(value, categories));
+  };
+
+  const validateStringOrNil = (value) => {
+    return R.or(R.isNil(value), R.is(String, value));
+  };
+  const findName = (name1, arr) => R.compose(
+    (value) => value.includes(name1),
```

Comment:
Tu peux utiliser `includes` de Ramda :
```suggestion
    R.includes,
```